pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -112,6 +112,12 @@ class StateVectorKokkos final
             }
         }
 
+#ifdef _WIN32
+        PL_ABORT_IF_NOT(num_qubits,
+                        "LightningKokkos zero-qubit device initialization is "
+                        "not supported on Windows.");
+#endif
+
```

Comment:
Does `default.qubit` support a zero-qubit state? I think that now that `default.qubit` does not need the number of qubits for initialization this is a question difficult to answer. But does it make sense for us to support a zero-size state vector? I mean, we could only abort for all cases.

Reply:
yaeh, `default.qubit` supports the zero-qubit state similar to `lightning.qubit` and `lightning.kokkos` (on Linux and macOS). This feature is also required in Catalyst. The zero-qubit support of `lightning.kokkos` on Windows, however, is more of an issue with the initialization of Kokkos state-vector of size 1, capacity 2 for 0-qubit. I could get a workaround for this, but as this is not a high priority now, we can create a PR with a proper & clean fix later. I added this to the changelog as well as tests in this PR for future reference. Anyway, I'll open a separate WIP PR later today for further discussions.